src/AudioManagerImpl.js

```diff
@@ -106,68 +67,55 @@ implements AudioManager {
                 "C3": "C6.wav",
                 "C4": "C6.wav",
                 "C5": "C6.wav",
-                "C6": "C6.wav",
+                "C6": "C6.wav"
             },
-            baseUrl: "/audio/left-turn/",
-        },
-        );
+            baseUrl: "/audio/left-turn/"
+        });
 
         this.samplers.left.connect(this.panner);
 
-        this.samplers.right = new Sampler(
-        {
+        this.samplers.right = new Sampler({
             urls: {
-            // The percussion instrument we used actually dooesn't vary it's pitch, we use the same sample at different
-            // pitches so that we can scale relative to the octave without ending up with wildy different tempos.
+                // The percussion instrument we used actually dooesn't vary it's pitch, we use the same sample at different
+                // pitches so that we can scale relative to the octave without ending up with wildy different tempos.
                 "C0": "C6.wav",
                 "C1": "C6.wav",
                 "C2": "C6.wav",
                 "C3": "C6.wav",
                 "C4": "C6.wav",
                 "C5": "C6.wav",
-                "C6": "C6.wav",
+                "C6": "C6.wav"
             },
-            baseUrl: "/audio/right-turn/",
-        },
-        );
+            baseUrl: "/audio/right-turn/"
+        });
 
         this.samplers.right.connect(this.panner);
 
-        this.samplers.movement = new Sampler(
-        {
+        this.samplers.movement = new Sampler({
             urls: {
                 "C0": "C0.wav",
                 "C1": "C1.wav",
                 "C2": "C2.wav",
                 "C3": "C3.wav",
                 "C4": "C4.wav",
                 "C5": "C5.wav",
-                "C6": "C6.wav",
+                "C6": "C6.wav"
             },
-            baseUrl: "/audio/long-bell/",
-        },
-        );
+            baseUrl: "/audio/long-bell/"
+        });
 
         this.samplers.movement.connect(this.panner);
     }
 
-    buildAnnouncementLookUpTable() {
-        this.announcementLookUpTable = {};
-        AnnouncementDefs.forEach(
-        (value, key) => {
-            const player = new Player(value);
-            player.toDestination();
-            this.announcementLookUpTable[key] = player;
-        },
-        );
-    }
-
-    playAnnouncement(soundName: AnnouncedSoundName) {
+    playAnnouncement(messageIdSuffix: string, intl: IntlShape, messagePayload: any) {
```

Comment:
Small mismatch with the type defined in `types.js`:

    playAnnouncement(messageIdSuffix: string, intl: IntlShape, messageOptions?: any)

`messagePayload` vs `messageOptions?`

Reply:
Fixed.